pkg/pillar/types/domainmgrtypes.go

```diff
@@ -25,6 +25,8 @@ type DomainConfig struct {
 	VifList           []VifInfo
 	IoAdapterList     []IoAdapter
 	CloudInitUserData string // base64-encoded
+	URL               string // rkt uses this URL to start the container
```

Comment:
same here -- rkt run doesn't use the URL, rkt run uses the container ID (based on sha256) that is a completely local connect

Reply:
On our internal communication, I was suggesting passing SHA256 to domain manager, to be inputted to rkt in issuing the run command. But then the group decided that URL can be used.
So this field has been added to DomainConfig structure.

Reply:
Well, these types of decisions have to be made in the open. Typically either on the mailing list or collaborating on shared documents.

At least I wasn't aware of this decision being made and I'm sure a lot of the community members are in the same situation. The idea for the code review is to uncover these hidden assumptions since we can't simply rely on design documents -- that will never work.

So I really think we shouldn't take a shortcut here and it seems that at least @nordmark agrees.